python_benchmarks/qr_codes/README.md

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
I propose to add intro phrase like:
"The benchmark aims to check QR codes detection and decoding with classic OpenCV implementation from objdetect module and dnn based implementation in opencv_contrib repo contributed by the WeChat team."